packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -62,6 +63,18 @@ export class CharacterCount extends GOVUKFrontendComponent {
   /** @private */
   maxLength = Infinity
 
+  /** @private */
+  configSchema = {
+    anyOf: [
+      {
+        required: ['maxwords']
```

Comment:
Probably need to put the error message alongside the validation rule?

Otherwise from a boolean `this.validateConfig()` how do we know we rule was broken?

```suggestion
        required: ['maxwords'],
        errorMessage: 'Either `maxlength` or `maxwords` must be provided'
```